test/__snapshot__/parser/module/top-level-await/export-var-await-expr-identifier.md

```diff
@@ -0,0 +1,280 @@
+                                                    "text": "foo",
+                                                    "rawText": "foo",
+                                                    "flags": 96,
+                                                    "start": 66,
+                                                    "end": 70
+                                                },
+                                                "flags": 32,
+                                                "start": 60,
+                                                "end": 70
+                                            },
+                                            "flags": 32,
+                                            "start": 56,
+                                            "end": 70
+                                        }
+                                    ],
+                                    "trailingComma": false,
+                                    "flags": 32,
+                                    "start": 56,
+                                    "end": 70
+                                },
+                                "flags": 32,
+                                "start": 54,
+                                "end": 72
+                            },
+                            "optionalToken": null,
+                            "type": null,
+                            "initializer": {
+                                "kind": 220,
+                                "propertyList": {
+                                    "kind": 218,
+                                    "properties": [],
+                                    "trailingComma": false,
+                                    "flags": 16,
+                                    "start": 76,
+                                    "end": 76
+                                },
+                                "flags": 48,
+                                "start": 74,
+                                "end": 77
+                            },
+                            "flags": 16,
+                            "start": 54,
+                            "end": 77
+                        }
+                    ],
+                    "flags": 16,
+                    "start": 54,
+                    "end": 77
+                },
+                "flags": 16,
+                "start": 50,
+                "end": 78
+            },
+            "namedExports": null,
+            "exportFromClause": null,
+            "fromClause": null,
+            "exportKind": 0,
+            "flags": 16,
+            "start": 43,
+            "end": 78
+        }
+    ],
+    "isModule": true,
+    "source": "var foo = 1;\n\nexport var name1 = await foo;\nexport var { x = await foo } = {};",
+    "fileName": "__root__",
+    "flags": 0,
+    "start": 0,
+    "end": 78
+}
+```
+
+### Printed
+
+```javascript
+
+var foo = 1;
+ var name1 = await foo;
+ var {
+  x = await foo
+} = {};
+```
```

Comment:
the printed code is incorrect.

Reply:
what is missing here?

Reply:
technically it's a valid code, but the input is:
```js
var foo = 1;
export var name1 = await foo;
export var { x = await foo } = {};
```

but the `export` is missing in the output.

Reply:
Printing issue again?